boards/mbed_lpc1768/system.c

```diff
@@ -382,34 +382,34 @@
 
 
 /* F_cco0 = (2 * M * F_in) / N  */
-#define __M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
-#define __N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
-#define __FCCO(__F_IN)    ((2ULL * __M * __F_IN) / __N)
-#define __CCLK_DIV        (((CCLKCFG_Val      ) & 0x00FF) + 1)
+#define M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
+#define N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
+#define FCCO(__F_IN)    ((2ULL * M * __F_IN) / N)
+#define CCLK_DIV        (((CCLKCFG_Val      ) & 0x00FF) + 1)
```

Comment:
`#define`-ing one letter names like `N` etc leads to incredibly difficult to understand compile problems when a new user writes their program to use the variable `N` for something.